### DISCUSSION TITLE
Promicro が片側のみ認識されない問題につきまして

### DISCUSSION THREAD

**Discussion by Kei1661** [Yowkees/keyball#349] [Q&A]
お世話になっております。
Keyball44を右手トラックボール仕様で組み立てておりましたところ、左手側はテストファームウェアの書き込み及びLEDの点灯確認までできたのですが、右手側をPCに接続しても認識されない状況です。ビルドガイドにありました通りリセットスイッチを2回押しても認識されませんでした。
また、Promicroに搭載されている赤いLEDがPC接続後しばらくすると消灯してしまいます。
![B69EB7F4-38E4-4478-B2E9-474A39DBBDAC](https://github.com/Yowkees/keyball/assets/145314326/84d8cda6-74a9-486d-849c-491177b1cf7f)

以下個人的に行ったテストの結果を記載させて頂きます。

・当初右手側に乗せて認識されなかったPromicroを左手側に乗せる→認識されファームウェアの書き込みも可能

・当初左手側に乗せ認識されたPromicroを右手側に乗せる→認識されない、Promicroの赤いLEDもしばらくして消灯

上記結果よりPromicroは正常なように見受けられますので、問題はPCB側かと思われますが素人なもので行き詰まっているところでございます。

お忙しい中恐縮ですが、問題解決の心あたりがございましたらご教示頂けると幸いです。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

左右のProMicroを入れ替えても右手側で問題があるということで、可能性が高いのが、
フルカラーLEDのどれかがVCCとGNDがショートした状態で壊れてしまった可能性です。

こちらの確認方法としては、１つずつLEDを取り外して問題のLEDを発見して交換する方法ですが、作業量が非常に多いです。
そのため、全LEDの故障を１度に確認するうまい方法があります。

下記写真の赤線で示した部分のパターンをカットすると、LEDすべてのVCCをProMicroから切り離すことができます。
赤線でしめすパターンが細くなっている部分をカッターの刃の背のとがった部分でカリカリ削ってパターンを除去してください。
もとに戻すにはジャンパーの要領で、ここに半田を盛ることで復活できます。

ひとまずこの方法で全LEDの電源を止めてProMicroが起動するかお試しいただければ幸いです。

以上、お手数ですがご対応よろしくお願いいたします。
